backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -259,7 +272,15 @@ public Optional<OrganizationRoleClaims> updateUserEmail(
     profile.setLogin(email);
     profile.setEmail(email);
     user.setProfile(profile);
-    user.update();
+    try {
+      user.update();
+    } catch (ResourceException e) {
+      if (e.getMessage()
```

Comment:
Same thing here - we should do something with the `ResourceException` in case it's not the one we're expecting.

Reply:
Done!! Thank you!! :D